src/com/amazon/ion/benchmark/DataConstructor.java

```diff
@@ -224,12 +228,78 @@ public static IonValue constructIonData(ReparsedType parsedTypeDefinition) {
                     return SYSTEM.newBlob(constructLobs(constraintMapClone));
                 case CLOB:
                     return SYSTEM.newClob(constructLobs(constraintMapClone));
+                case STRUCT:
+                    return constructIonStruct(constraintMapClone);
+                case LIST:
+                    return constructIonList(constraintMapClone);
```

Comment:
Are you planning to add `IonSexp` construction in a separate PR? Asking this here because `ordered_elements` can also be used for constructing `IonSexp`.

Reply:
> Are you planning to add `IonSexp` construction in a separate PR? Asking this here because `ordered_elements` can also be used for constructing `IonSexp`.

Yes , IonSexp generation is planning to be added in the next PR.